src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableOrchestrationClient.cs

```diff
@@ -261,6 +261,16 @@ public interface IDurableOrchestrationClient
         /// <returns>Returns a task which completes when the status has been fetched.</returns>
         Task<DurableOrchestrationStatus> GetStatusAsync(string instanceId, bool showHistory = false, bool showHistoryOutput = false, bool showInput = true);
 
+        /// <summary>
+        /// Get the status of multiple instances.
+        /// </summary>
+        /// <param name="instanceIds"> List of multiple instance IDs that needed to check that status.</param>
+        /// <param name="showHistory">Boolean marker for including execution history in the response.</param>
+        /// <param name="showHistoryOutput">Boolean marker for including input and output in the execution history response.</param>
+        /// <param name="showInput">If set, fetch and return the input for the orchestration instance.</param>
+        /// <returns>Returns a list of orchestration status values.</returns>
```

Comment:
spelling suggestions:
```suggestion
        /// <summary>
        /// Get the status of multiple instances.
        /// </summary>
        /// <param name="instanceIds">The instanceIDs to query.</param>
        /// <param name="showHistory">Whether to include the instance history in the response.</param>
        /// <param name="showHistoryOutput">Whether to include the history output in the response.</param>
        /// <param name="showInput">Whether to include the input of the orchestration instance.</param>
        /// <returns>Returns a list of orchestration statuses.</returns>
```